src/topology/algebra/infinite_sum.lean

```diff
@@ -452,6 +452,13 @@ lemma tsum_eq_single {f : β → α} (b : β) (hf : ∀b' ≠ b, f b' = 0)  :
   ∑'b, f b = f b :=
 (has_sum_single b hf).tsum_eq
 
+lemma tsum_tsum_eq_single (f : β → γ → α) (b : β) (c : γ)
+  (hf : ∀ (b' : β) (c' : γ), b ≠ b' ∨ c ≠ c' → f b' c' = 0) :
```

Comment:
I'm not sure if it's better to have this one assumption or two assumptions, one with `b ≠ b'` and another one with `c ≠ c'`. Otherwise LGTM.

Reply:
I think the two is actually probably easier to work with, avoids needing some kind of induction in proving the assumption, so I switched it to that